### PR TITLE
Update mitigation-serialization-control-characters.md

### PR DESCRIPTION
**Pull request by JasonWHowell**
Shortening title as per SEO Audit item 1656857

Title truncates in google searches
image

### CODE REVIEW

File: docs/framework/migration-guide/mitigation-serialization-control-characters.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 ---
-title: "Mitigation: Serialization of Control Characters with the DataContractJsonSerializer"
-ms.date: "04/07/2017"
+title: "Mitigation: Serialize control characters - DataContractJsonSerializer"
```
Comment (mairaw) [inline docs/framework/migration-guide/mitigation-serialization-control-characters.md]: I don't think this change is correct.

Comment (mairaw) [reply docs/framework/migration-guide/mitigation-serialization-control-characters.md]: How long can titles be? Maybe:

```suggestion
title: "Serialization of control characters with DataContractJsonSerializer"
```

Comment (gewarren) [reply docs/framework/migration-guide/mitigation-serialization-control-characters.md]: Or just "Serialize control characters with DataContractJsonSerializer"?

Comment (mairaw) [reply docs/framework/migration-guide/mitigation-serialization-control-characters.md]: I just find the active verb here wrong. This is about the behavior with that control.

**Review by gewarren — APPROVED** [review]

**Pull request merged**